Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -165,16 +146,14 @@ - (void)handleRequest:(BTPaymentFlowRequest *)request
             return;
         }
 
-        if (configuration.cardinalAuthenticationJWT && self.versionRequested == BTThreeDSecureVersion2) {
+        if (configuration.cardinalAuthenticationJWT) {
             [self prepareLookup:apiClient completion:^(NSError * _Nullable error) {
                 if (error != nil) {
                     [delegate onPaymentComplete:nil error:error];
                 } else {
                     [self startRequest:request configuration:configuration];
                 }
             }];
-        } else {
-            [self startRequest:request configuration:configuration];
         }
```

Comment:
Since there is no more `else` here, I think we need to handle the case of no `configuration.cardinalAuthenticationJWT` and return some error to the merchant, vs right now this no-ops.

Reply:
+1

Reply:
Good catch - updated: https://github.com/braintree/braintree_ios/pull/979/commits/fce558979ad35575c473a4de74d2d0ae63271c04